app/controllers/concerns/sufia/breadcrumbs.rb

```diff
@@ -27,10 +27,10 @@ def trail_from_referer
 
     def add_breadcrumb_for_controller
       case controller_name
-      when 'file_sets'.freeze, 'my/files'.freeze, 'batch_edits'.freeze
-        add_breadcrumb I18n.t('sufia.dashboard.my.files'), sufia.dashboard_files_path
+      when 'file_sets'.freeze, 'my/works'.freeze, 'batch_edits'.freeze
+        add_breadcrumb I18n.t('sufia.dashboard.my.works'), sufia.dashboard_works_path
       when 'stats'.freeze
```

Comment:
Could we add `'stats'.freeze` to line 30 instead of keeping an extra branch that calls the same action as the prior condition?